py3-cairo will fail pkgconf test pipeline

If the pkgconf test pipeline is added to py3 it will fail in the following way:

```
2024/11/12 23:14:33 INFO running step "test/pkgconf"
2024/11/12 23:14:33 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:14:33 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:14:33 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:14:33 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:14:33 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN + basename /home/build/melange-out/py3-cairo-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN + dev_pkg=py3-cairo-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN + apk info -L py3-cairo-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN + grep -q ^Name: usr/lib/pkgconfig/py3cairo.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN + basename usr/lib/pkgconfig/py3cairo.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN + lib_name=py3cairo uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN + echo usr/lib/pkgconfig/py3cairo.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN + grep -q '^usr/lib/pkgconfig/py3cairo.pc$\|^usr/share/pkgconfig/py3cairo.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN + pkgconf --exists py3cairo uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN + pkgconf --print-errors --exists py3cairo uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN Package cairo was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN Perhaps you should add the directory containing `cairo.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:33 WARN Package 'cairo', required by 'py3cairo', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:34 INFO pod 03d9c82e4f3a4fe3a5961ea139afd0947f241591191990585cc9e6e662ee7e07 terminated
2024/11/12 23:14:34 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:14:34 INFO   workspace dir: /home/user/tmp/melange-workspace-1619604427
2024/11/12 23:14:34 INFO   guest dir: /home/user/tmp/melange-guest-1845162078
2024/11/12 23:14:34 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/py3-cairo] Error 1
```

--- a/py3-cairo.yaml
+++ b/py3-cairo.yaml
@@ -1,7 +1,7 @@
 package:
   name: py3-cairo
   version: 1.27.0
-  epoch: 1
+  epoch: 2
   description: Python3 bindings for the cairo graphics library
   copyright:
     - license: LGPL-2.0-or-later
@@ -118,11 +118,13 @@ subpackages:
     description: py3-cairo dev
     dependencies:
       runtime:
+        - cairo-dev
         - py3-cairo
     pipeline:
       - uses: split/dev
     test:
       pipeline:
+        - uses: test/pkgconf
         - uses: py/one-python
           with:
             content: |

